DFC.ServiceTaxonomy.Editor/Recipes/Service Taxonomy.recipe.json

```diff
@@ -1507,6 +1590,115 @@
                     },
                     "ContentPartFieldDefinitionRecords": []
                 },
+                {
+                    "Name": "ApprenticeshipStandard",
+                    "Settings": {},
+                    "ContentPartFieldDefinitionRecords": [
+                        {
+                            "FieldName": "TextField",
+                            "Name": "Reference",
+                            "Settings": {
+                                "ContentPartFieldSettings": {
+                                    "DisplayName": "Apprenticeship Reference",
+                                    "Position": "0"
+                                }
+                            }
+                        },
+                        {
+                            "FieldName": "NumericField",
+                            "Name": "MaximumFunding",
+                            "Settings": {
+                                "ContentPartFieldSettings": {
+                                    "DisplayName": "Maximum Funding Amount (£)",
+                                    "Position": "1"
+                                },
+                                "NumericFieldSettings": {
+                                    "Scale": 1
```

Comment:
Seems odd to store £ amounts to 1 decimal place. Either whole number or 2 decimal places makes more sense intuitively.

Reply:
They are whole pounds so I can change this to whole number